### PR TITLE
Workshop 5

### PR DESCRIPTION
**Pull request by serhatgemici**

### CODE REVIEW

File: src/main/java/serhat/workshop_5/CatalogPage.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Override
+    protected void openPage() {
+
+        if (!partialURL.equals("/cars") && !partialURL.equals("/motorcycles") && !partialURL.equals("/commercial-vehicles") && !partialURL.equals("/all")) {
+            System.out.println("Opening URL: " + BASE_URL + partialURL);
+            System.out.println("Error: Cannot handle given partial url /sellcar, page will not be shown");
+        }
+
+        if (partialURL == "/cars") {
```
Comment (tgrablow) [inline src/main/java/serhat/workshop_5/CatalogPage.java]: Please, use equals()
Example: http://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java

File: src/main/java/serhat/workshop_5/CatalogPage.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            System.out.println("Opening URL: " + BASE_URL + partialURL);
+            showHeader();
+            System.out.println("cars catalog is shown");
+            showFooter();
+            System.out.println("***\n");
+        } else if (partialURL == "/motorcycles") {
```
Comment (tgrablow) [inline src/main/java/serhat/workshop_5/CatalogPage.java]: Please, use equals()
Example: http://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java

File: src/main/java/serhat/workshop_5/CatalogPage.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            System.out.println("Opening URL: " + BASE_URL + partialURL);
+            showHeader();
+            System.out.println("motorcycles catalog is shown");
+            showFooter();
+            System.out.println("***\n");
+        } else if (partialURL == "/commercial-vehicles") {
```
Comment (tgrablow) [inline src/main/java/serhat/workshop_5/CatalogPage.java]: Please, use equals()
Example: http://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java

File: src/main/java/serhat/workshop_5/CatalogPage.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            System.out.println("Opening URL: " + BASE_URL + partialURL);
+            showHeader();
+            System.out.println("commercial-vehicles catalog is shown");
+            showFooter();
+            System.out.println("***\n");
+        } else if (partialURL == "/all") {
```
Comment (tgrablow) [inline src/main/java/serhat/workshop_5/CatalogPage.java]: Please, use equals()
Example: http://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java

File: src/main/java/serhat/workshop_5/LoginPage.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.sun.xml.internal.rngom.parse.host.Base;
+
+/**
+ * Created by serhatgemici on 04.04.17.
+ */
+public class LoginPage extends BasePage {
+
+    public LoginPage(String partialURL) {
+
```
Comment (tgrablow) [inline src/main/java/serhat/workshop_5/LoginPage.java]: Empty line

File: src/main/java/serhat/workshop_5/LoginPage.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        System.out.println("login form is shown");
+    }
+
+    @Override
+    public void openPage() {
+        showHeader();
+        showLoginForm();
+        showFooter();
+        System.out.println("***");
+
```
Comment (tgrablow) [inline src/main/java/serhat/workshop_5/LoginPage.java]: Empty line

File: src/main/java/serhat/workshop_5/BasePage.java [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+    protected static final String BASE_URL = "http://www.buycarshere.com";
+    protected static String partialURL;
+
+    protected BasePage(String partialURL) {
+        this.partialURL = partialURL;
+    }
+
+    protected void showHeader() {
+
```
Comment (tgrablow) [inline src/main/java/serhat/workshop_5/BasePage.java:19]: Empty lines here and below

**Review by tgrablow — CHANGES_REQUESTED** [review]
Just minor issues

**Review by serhatgemici — COMMENTED** [review]
== operators changed to .equals()

**Review by serhatgemici — COMMENTED** [review]
empty lines removed

**Review by serhatgemici — COMMENTED** [review]
empty lines removed

**Review by tgrablow — APPROVED** [review]

**Pull request merged**